bin/update-changelog.js

```diff
@@ -7,10 +7,11 @@ const readline = require('readline');
 const minimist = require('minimist');
 
 const argv = minimist(process.argv, {
-  boolean: ['dry-run', 'help'],
+  boolean: ['dry-run', 'help', 'hide-reviewer'],
   string: ['since'],
   default: {
-    'dry-run': false
+    'dry-run': false,
+    'hide-reviewer': false
```

Comment:
Ich war leider etwas zu schnell bei mergen.
Kannst du den Parameter noch in die `README.md` mit aufnehmen?